ontopy/ontology.py

```diff
@@ -1223,6 +1226,8 @@ def new_entity(self, name: str, parent: ThingClass) -> ThingClass:
                 f"Error in label name definition {name}: "
                 "Label consists of more than one word."
             )
-        with self:
-            entity = types.new_class(name, (parent,))
+        parent = parent if isinstance(parent, list) else [parent]
+        for thing in parent:
+            with self:
+                entity = types.new_class(name, (thing,))
```

Comment:
This will create one new entity for each parent, which is not what we want. Simply replace the for loop with

```python
with self:
    entity = types.new_class(name, parents)
```

Reply:
It will not, but I agree it is more elegant to do it in a oneliner, need to make parents a tuple.